### PR TITLE
KFP Upgrade

### PR DESCRIPTION
**Pull request by eedorenko**
- Installs Kubeflow Pipelines 1.0.0
- Installs Istio 1.6.5 
- Fixes Kubeflow Pipelines cache installation
- Fixes Seldon Core installation on K8s v 1.18

### CODE REVIEW

File: docs/Kubeflow-install.md [hunk 1]
```diff
@@ -13,35 +13,6 @@
 ## Install Kubeflow core + Kubeflow pipelines
 
 * Use [kubeflow-install.sh](../setup/kfp/kubeflow-install.sh) script or follow the manual steps below
```
Comment (jotaylo) [inline docs/Kubeflow-install.md:15]: nit: Remove "or follow the manual steps below"

File: setup/seldon/setup.sh [hunk 2]
```diff
@@ -1,5 +1,9 @@
 kubectl create namespace seldon-system
 
-helm install seldon-core seldon-core-operator --repo https://storage.googleapis.com/seldon-charts --set istio.enabled=true --set usageMetrics.enabled=true --namespace seldon-system
+# Need to create seldon crd explicitly (not with a helm chart)
+# due to incompatibility issue with K8s v1.18
```
Comment (jotaylo) [inline setup/seldon/setup.sh:4]: any more info about this incompatibility? (e.g. error message or link to issue)

File: docs/Kubeflow-install.md (hunk 1 above)
Comment (eedorenko) [reply docs/Kubeflow-install.md]: Done

File: setup/seldon/setup.sh (hunk 2 above)
Comment (eedorenko) [reply setup/seldon/setup.sh:4]: Added a link

**Pull request merged**